### ISSUE TITLE
Semantic tutorial

### ISSUE THREAD

**Issue by Garito** [vwall/compass-twitter-bootstrap#74]
Hi!
I recently found compass and sass because I don't use rails on my workflow but I'm pretty interested in using TB in a semantic way

So, now I have installed compass-twitter-bootstrap and I can use it in my projects (I've try semantic layout more or less and buttons)

My next step is use it with tabs but I can't understand the way to do that

I've tried something like this:

HTML

``` html
<nav>
    <ul>
        <li><a href="#">Menu Option 1</a></li>
        <li><a href="#">Menu Option 2</a></li>
        <li><a href="#">Menu Option 3</a></li>
    </ul>
</nav>
```

SCSS

``` sass
nav {
    @extend .nav;
    @extend .nav-tabs;
}
```

but I don't have any tab (I only see the bottom line used in the TB tabs)

What am I missing?

Is this library what I want? (I mean if I could achieve my semantic needs with it or I need to search for a better solution)

Thanks a lot!!
Sorry for the advanced guys in the list pissed of with newbies questions

**Comment by kristianmandrup** [comment]
I started a project aimed at making it easier to use Twitter Bootstrap in a more semantic fashion. You are most welcome to experiment with it and help out improving it ;)

https://github.com/kristianmandrup/semantic-sass-bootstrap

**Comment by Garito** [comment]
Jajajajaja, sincronicity! I have your project page opened before I try compass-twitter-bootstrap but, again, I'm not a ruby user so I don't have a clue about how to install it

Could you point me how to install it in a mountain lion (with gem) and compass installed?

I have problems to find my Gemfile to add this line:

gem 'semantic-sass-bootstrap', git: 'git://github.com/kristianmandrup/semantic-sass-bootstrap.git'

Thanks a lot!

PS: do you prefer I translate this question to the other project?

**Comment by kristianmandrup** [comment]
Try using rails_apps_composer to create a Rails app, then insert the gemlggenerated generated Gemfile of your app and run $ bundle to install it. Also read the free online Rails tutorial. Good luck ;)

**Comment by Garito** [comment]
Sorry Kristian but I won't to spend time in a technology only to use a suppossed agnostic one like sass and compass (even if it is as integrated to ruby as this one)

Question: can I copy the files in a concrete folder in order to use it?

**Comment by kristianmandrup** [comment]
Yes, you can just copy the files over n case you don'w wanna use it with sprockets (asset pipeline).

**Comment by Garito** [comment]
Yeah! I'm making some experiments!
But I don't catch something apparently

I have a normal compass project with the sass and the stylesheets folder
In the sass one I copy all the scss files you provide in order to avoid any path problem
Now I import the bootstrap scss file you provide (the one with all the rest imported)

Trying to compile it with compass watch raises this error:

error sass/semantic_bootstrap.scss (Line 52 of sass/_buttons.scss: Invalid CSS after "  @if $size ": expected "{", was "= 'large' {")

Could you point me what's the problem?

Thanks a lot!!
